prac_05/state_names.py

```diff
@@ -0,0 +1,24 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+
+for code, states in CODE_TO_NAME.items():
+    print(f"{code:<3} is {CODE_TO_NAME[code]}")
+
+
+state_code = input("Enter short state: ")
+while state_code != "":
+    state_code = state_code.upper()
+    try:
+        print(f"{state_code} is {CODE_TO_NAME[state_code]}")
+    except:
```

Comment:
Might be more beneficial to except for a specific error code rather than a catch-all.